frontend/src/metabase/query_builder/components/CompareAggregations/components/ColumnPicker/ColumnPicker.tsx

```diff
@@ -0,0 +1,104 @@
+import type { ComponentPropsWithoutRef } from "react";
+import { forwardRef, useCallback, useMemo } from "react";
+import { t } from "ttag";
+
+import { Checkbox, Flex, MultiSelect, Text } from "metabase/ui";
+
+import type { ColumnType } from "../../types";
+
+import S from "./ColumnPicker.module.css";
+
+interface ItemType {
+  example: string;
+  isSelected: boolean;
+  label: string;
+  value: ColumnType;
+}
+
+interface Props {
+  value: ColumnType[];
+  onChange: (value: ColumnType[]) => void;
+}
+
+const COLUMN_OPTIONS: Omit<ItemType, "isSelected">[] = [
+  {
+    example: "1826, 3004",
+    label: t`Previous value`,
+    value: "offset",
+  },
+  {
+    example: "+2.3%, -0.1%",
+    label: t`Percentage difference`,
+    value: "percent-diff-offset",
+  },
+  {
+    example: "+42, -3",
+    label: t`Value difference`,
+    value: "diff-offset",
+  },
+];
+
+export const ColumnPicker = ({ value, onChange }: Props) => {
+  const handleChange = useCallback(
+    (values: string[]) => {
+      onChange(values as ColumnType[]);
+    },
+    [onChange],
+  );
+
+  const options = useMemo(() => {
+    return COLUMN_OPTIONS.map(option => ({
+      ...option,
+      isSelected: value.includes(option.value),
+    }));
+  }, [value]);
+
+  return (
+    <MultiSelect
+      label={t`Columns to create`}
+      data={options}
+      disableSelectedItemFiltering
+      itemComponent={Item}
+      placeholder={t`Columns to create`}
+      rightSection={null}
+      styles={{
+        item: {
```

Comment:
I had problems with the specificity of these overrides:
- I could not do them via `.module.css` file without `!important`
- I had to list all 4 cases

Reply:
Have designers approved this hack?

Reply:
Yes, it was requested (see [Slack](https://metaboat.slack.com/archives/C0645JP1W81/p1717179784159309?thread_ts=1716988161.943959&cid=C0645JP1W81))